libcomposefs/lcfs-mount.c

```diff
@@ -666,6 +668,35 @@ static errint_t lcfs_mount(struct lcfs_mount_state_s *state)
 	return -EINVAL;
 }
 
+static int lcfs_mount_copy_and_seal(int fd) {
+	int memfd = memfd_create(CFS_MOUNT_SOURCE, MFD_CLOEXEC | MFD_ALLOW_SEALING);
+	if (memfd < 0)
+		return -errno;
+
+	// in the general case, copy_file_range() to a memfd doesn't work
+	ssize_t n;
+	do {
+		char buffer[1 << 20];  // 1MB
+		n = read(fd, buffer, sizeof buffer);
```

Comment:
Need to handle EINTR.

Reply:
We already have `copy_file_data_classic()`